quanto/tensor/optimizers/mse_optimizer.py

```diff
@@ -0,0 +1,72 @@
+from typing import Tuple, Union
+
+import torch
+from torch import Tensor
+
+from quanto.tensor.core import axis_to_dim
+
+from .affine_optimizer import AffineOptimizer
+from .symmetric_optimizer import SymmetricOptimizer
+
+
+def _fake_quantize(base: Tensor, scale: Tensor, zeropoint: Union[int, Tensor], qmin: int, qmax: int) -> Tensor:
```

Comment:
Please reuse the actual quantizers to make usre the quantizing algorithms are identical.